README.md

```diff
@@ -1,7 +1,8 @@
 # Horton
-A repository for doing shared R&amp;D on Sufia for the Development team.
+A repository for doing shared R&amp;D on Hyrax for the Development team.
```

Comment:
@mcritchlow Yes, It sounds good and I've corrected it. See commit https://github.com/ucsdlib/horton/pull/38/commits/1d28f7d394707b395b5c49ef5a7077f28f1f3795.